### PR TITLE
Fix: nginx reload after cert update

### PR DESCRIPTION
**Pull request by till**
# Changes

Bugfix: cron script only supported haproxy before

Related: easybib/ops#207

### CODE REVIEW

File: ies-letsencrypt/templates/default/certbot-cronjob.sh.erb [hunk 1]
```diff
@@ -14,6 +14,15 @@ COMBINED="<%=@ssl_dir%>/cert.combined.pem"
 
 CERT_FILE="$etc/live/$DOMAIN/fullchain.pem"
 KEY_FILE="$etc/live/$DOMAIN/privkey.pem"
+NGINX_INIT="/etc/init/nginx.conf"
```
Comment (till) [inline ies-letsencrypt/templates/default/certbot-cronjob.sh.erb]: Assumption, this file is available - we never have both installed currently.

File: nginx-app/templates/default/redirect-ssl.conf.erb [hunk 2]
```diff
@@ -1,5 +1,6 @@
 server {
-    listen 443;
+    listen 80;
```
Comment (till) [inline nginx-app/templates/default/redirect-ssl.conf.erb]: Make them listen on both ports so we don't have too many configs to dig through. Also certbot challenge needs port 80 to verify.

File: ies-letsencrypt/templates/default/certbot-cronjob.sh.erb (hunk 1 above)
Comment (gilleyj) [inline ies-letsencrypt/templates/default/certbot-cronjob.sh.erb]: agreed.

File: nginx-app/templates/default/redirect-ssl.conf.erb (hunk 2 above)
Comment (gilleyj) [inline nginx-app/templates/default/redirect-ssl.conf.erb]: agreed

**Pull request merged**